source/ed/ed_persistor.cpp

```diff
@@ -1045,21 +1055,21 @@ void EdPersistor::insert_vehicle_journeys(const std::vector<types::VehicleJourne
     }
     this->lotus.finish_bulk_insert();
     for (types::VehicleJourney* vj : vehicle_journeys) {
-        std::string values = "";
+        std::string values;
         if (vj->prev_vj) {
-            values = "previous_vehicle_journey_id = " + boost::lexical_cast<std::string>(vj->prev_vj->idx);
+            values = "previous_vehicle_journey_id = " + std::to_string(vj->prev_vj->idx);
```

Comment:
Are you really sure about that ? is it actually the same thing ?

Reply:
I can check that just in case!

Reply:
`vj->prev_vj->idx` is a uint_32_t so it works! :smile: